packages/binaryen/build.sh

```diff
@@ -0,0 +1,7 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/WebAssembly/binaryen
+TERMUX_PKG_DESCRIPTION="Binaryen is a compiler and toolchain infrastructure library for WebAssembly"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_VERSION=version_98
+TERMUX_PKG_SRCURL=https://github.com/WebAssembly/binaryen/archive/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=9f805db6735869ab52cde7c0404879c90cf386888c0f587e944737550171c1c4
+TERMUX_PKG_BUILD_IN_SOURCE=true
```

Comment:
Is this required, have you tried without?

Reply:
I just try and it's not needed